### ISSUE TITLE
version update and do compatible with stream_chat_flutter

### ISSUE THREAD

**Issue by jawad-arshad** [GetStream/stream-feed-flutter#258]
**Describe the bug**

**What version of Flutter do you use?**

**What package are you using? What version?**

**What platform is it about?**
- [x] Android
- [x] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS
- [ ] Linux

**a copy of `flutter doctor --verbose`**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Logs **
Run `flutter analyze` and attach any output of that command below.
If there are any analysis errors, try resolving them before filing this issue.

Paste the output of running `flutter doctor -v` here.

**Additional context**

**Comment by jawad-arshad** [comment]
please upgrade your package I'm using your both packages in one project when ever I try to upgrade chat package it says me dio is not compatible with feeds dio verison so please upgrade your feed package and make sure its compatible with latest streams chat package thanku

**Comment by pitch15solutions** [comment]
Yes -- current stream_chat_flutter (9.0.0) NOT compatible with latest stream_feed (0.6.0+2)

**Comment by zulu-tech-partners** [comment]
> Yes -- current stream_chat_flutter (9.0.0) NOT compatible with latest stream_feed (0.6.0+2)

so what's the solution for that?  i wanna use both stream_chat and stream_feed in one app without compatibility issue

**Comment by aShrestha-Outcode** [comment]
@zulu-tech-partners  did you find any solution to this.  I am getting similar issue

`Because stream_feed_flutter_core >=0.4.1 depends on dio ^4.0.1 and hype_mobile depends on dio 5.7.0, stream_feed_flutter_core >=0.4.1 is forbidden.
So, because hype_mobile depends on stream_feed_flutter_core 0.8.0, version solving failed.`
